### PR TITLE
feat(ui): Park/Unpark Escorts in System

### PR DESCRIPTION
**Pull request by Koranir**
**Feature:** This PR implements the feature request detailed and discussed in issue #7541 

## Feature Details
Adds a button in the info panel to park/unpark all ships in system

## UI Screenshots

https://user-images.githubusercontent.com/101683475/200823187-77c02330-aedb-46ef-b7be-bd7827300857.mp4

## Performance Impact
N/A

### CODE REVIEW

**Review by Hurleveur — APPROVED** [review]
Looks good to me code wise

**Comment by Zireael07** [comment]
Does that include the Unpark All keybind name change?

**Comment by Hurleveur** [comment]
It wasnt changed in this PR?

**Comment by Zireael07** [comment]
I was asking because in the issue, people were suggesting the keybind name be changed to reduce misunderstandings

**Comment by Koranir** [comment]
It's still a useful button, especially when usesd in conjunction with the new button, as was shown in the video.

**Comment by Zireael07** [comment]
Of course it's useful, I was just saying that it could be named better according to the issue linked

**Comment by Koranir** [comment]
Unpark All does exactly what it says it does though?

There was a request to change the behaviour of the button, but the button is still useful so I added a new button that does it instead.

**Comment by Hurleveur** [comment]
What about naming it Park In System? Seems clearer to me than Park Close
I think n is free? Or y?

**Comment by Koranir** [comment]
Prk in system is too long for the button

**Comment by 1010todd** [comment]
Park Local?

File: source/PlayerInfoPanel.cpp [hunk 1]
```diff
@@ -219,16 +219,25 @@ void PlayerInfoPanel::Draw()
 	if(panelState.CanEdit() && panelState.Ships().size() > 1)
 	{
 		bool allParked = true;
+		bool allParkedSystem = true;
+		bool hasParked = false;
```
Comment (quyykk) [inline source/PlayerInfoPanel.cpp:223]: `hasParked` is an unused variable FYI.

**Review by quyykk — APPROVED** [review]
Thanks!

**Pull request merged**